CMakeLists.txt

```diff
@@ -520,6 +528,10 @@ else()
     set(DILIGENT_INSTALL_PDB OFF)
 endif()
 
+if(PLATFORM_FREEBSD AND EXISTS /usr/local/include)
+    target_include_directories(Diligent-BuildSettings SYSTEM AFTER INTERFACE /usr/local/include)
```

Comment:
I tried to find some info about this problem, and I think that the right way to fix this is to set `CMAKE_REQUIRED_INCLUDES` at your project's CMake level:

https://github.com/dotnet/runtime/issues/4073

Reply:
Did you try setting the `CMAKE_REQUIRED_INCLUDES` in your project CMake?